SharedPackages/UIComponents/Package.resolved

```diff
@@ -0,0 +1,15 @@
+{
+  "originHash" : "9a9153fd0315e8d8297f1d738e08612405ff9980556d89b2e9841c568e9a4c38",
+  "pins" : [
+    {
+      "identity" : "designresourceskit",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/duckduckgo/DesignResourcesKit",
```

Comment:
Unrelated to this PR, but I'm increasingly convinced that we should move DesignResourcesKit into the monorepo as well, since quite a few packages depend on it and I could see it being a pain to update as a result. I'll get a discussion task up about that.